### PR TITLE
clone less

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: src/librustdoc/passes/lint/html_tags.rs [hunk 1]
```diff
@@ -25,91 +25,85 @@ pub(crate) fn visit_item(cx: &DocContext<'_>, item: &Item) {
                 Some(sp) => sp,
                 None => item.attr_span(tcx),
             };
-            tcx.struct_span_lint_hir(
-                crate::lint::INVALID_HTML_TAGS,
-                hir_id,
-                sp,
-                msg.to_string(),
```
Comment (matthiaskrgr) [inline src/librustdoc/passes/lint/html_tags.rs:32]: remove this .to_string() here, rest is formatting

Comment (fbstj) [reply src/librustdoc/passes/lint/html_tags.rs:32]: https://github.com/rust-lang/rust/pull/117540/files?diff=unified&w=1 helps

**Comment by fmease** [comment]
Love the branch name 😂😂

**Comment by fmease** [comment]
Thanks for approving, I didn't have enough time today.

**Comment by nnethercote** [comment]
Did you find these manually?

**Comment by matthiaskrgr** [comment]
I used clippy :)

**Comment by nnethercote** [comment]
I am still very unclear about the status of using clippy on rustc...

**Comment by matthiaskrgr** [comment]
Yeah, sometimes it breaks, sometimes it ICES, sometimes you have to manually hack around in x.py to get it to work, its definitely not a tier 1 `x.py` command :smile:

**Comment by est31** [comment]
> Thanks for approving, I didn't have the time for reviews today.

Seeing #117537 I just went over the list of open PRs and approved the ones that seemed easy enough to get merged. I hope this lessens the burden a little (279 PRs waiting on review, wow).

**Comment by matthiaskrgr** [comment]
seems to be the same noise as in https://github.com/rust-lang/rust/pull/115333/files (a comment-only pr)

**Comment by pnkfelix** [comment]
* the regressions here seem to be correlated with the same spike of noise that afflicted #115333.
* there is more [discussion of why on zulip](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra/topic/perf.20is.20slow.20again/near/400329622)
* marking as triaged

@rustbot label: +perf-regression-triaged

**Pull request merged**